src/core/agent/services/connectionService.ts

```diff
@@ -60,9 +58,7 @@ class ConnectionService extends AgentService {
     // This will take our first KERI identifier and get the server to resolve it, so that the connection is resolved from both sides and we can issue to this wallet using its API.
     if (url.includes("dev.keria.cf-keripy.metadata.dev.cf-deployments.org")) {
       // This is inefficient but it will change going forward.
-      const aid = (await Agent.agent.identifiers.getIdentifiers()).find(
-        (identifier) => identifier.method === IdentifierType.KERI
-      );
+      const aid = (await Agent.agent.identifiers.getIdentifiers())[0];
```

Comment:
If there are no identifiers, this will throw an error for index out of bounds access